src/components/scenes/GettingStartedScene.tsx

```diff
@@ -88,23 +82,17 @@ const slide4 = {
   title: lstrings.getting_started_slide_4_title
 }
 
-const sectionsVariantMap: { [key: string]: SectionData[] } = {
-  default: [slide1, slide2, slide3, slide4],
-  C_UspsMinusWGYC: [slide2, slide3, slide4],
-  D_UspsAltWGYC: [slide1Alt, slide2, slide3, slide4]
-}
-
 export const GettingStartedScene = (props: Props) => {
   const { navigation, route } = props
   const dispatch = useDispatch()
   const { experimentConfig } = route.params
-  const { createAccountType, landingType } = experimentConfig
+  const { createAccountType } = experimentConfig
   const context = useSelector(state => state.core.context)
   const isLoggedIn = useSelector(state => state.ui.settings.settingsLoaded ?? false)
   const localUsers = useWatch(context, 'localUsers')
   const hasLocalUsers = localUsers.length > 0
 
-  const sections: SectionData[] = sectionsVariantMap[landingType] ?? sectionsVariantMap.default
+  const sections: SectionData[] = [slide1, slide2, slide3, slide4]
```

Comment:
Please move this array outside the component, up above where the slides themselves are defined.